src/Microsoft.ApplicationInsights/Extensibility/Implementation/CorrelationLookup/CorrelationIdHelper.cs

```diff
@@ -0,0 +1,70 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.Implementation.CorrelationLookup
+{
+    using System.Globalization;
+    using System.Text.RegularExpressions;
+
+    internal static class CorrelationIdHelper
+    {
+        private const string CorrelationIdFormat = "cid-v1:{0}";
```

Comment:
`cid-v1` is a special string that describes that ID was taken from AI backend. I haven't looked thru the code. Make sure it is not hardcoded for static and dictionary providers. So we will enable other types of correlation ID to be defined going forward.

Reply:
Thank you for the clarification!

Reply:
@SergeyKanzhelev if the DictionaryProvider should return an unformatted ApplicationId, should the Interface method be renamed to TryGetApplicationId ?

Counter argument is that our ApplicationInsightsProvider **WILL** return a formatted id so it may be inappropriate to name it an ApplicationId.

Do we need a different vocabulary here?

Reply:
I think entire string `cid-v1:XXXXXXXX` is an application ID. It's like person identity can be established by `ssn:XXX` and `driver-lic-id:XXXX`.